app/controllers/admin/dashboard_controller.rb

```diff
@@ -1,7 +1,15 @@
-class Admin::DashboardController < ApplicationController
+class Admin::DashboardController < Admin::BaseController
```

Comment:
This edit is simply adding the Admin::BaseController as the parent class of the AdminDashboard.  It's so awesome being able to see inheritance working at full throttle.